index.bs

```diff
@@ -942,12 +942,29 @@ spec:css-syntax-3;
 
     1.  Let |p| be [=a new promise=].
 
-    1.  Let |origin| be the [=current settings object=]'s [=environment settings object/origin=].
+    1. For each |interface| in |options|' <a>relevant credential interface objects</a>:
+
+        1. If |settings|' [=active credential types=] [=set/contains=] |interface|'s
+            {{Credential/[[type]]}}, return [=a promise rejected with=] a "{{NotAllowedError}}"
```

Comment:
Unrelated to the PR but can you file a crbug for this & to have a corresponding WPT?

Reply:
Filed https://bugs.chromium.org/p/chromium/issues/detail?id=1365979